tests/EdFi.Ods.Api.IntegrationTests/StaffUSIToEducationOrganizationIdAuthViewTests.cs

```diff
@@ -0,0 +1,348 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using NUnit.Framework;
+
+namespace EdFi.Ods.Api.IntegrationTests
+{
+    [TestFixture]
+    public class StaffUSIToEducationOrganizationIdAuthViewTests : DatabaseTestFixtureBase
```

Comment:
Could you please change it from 'StaffUSI' to 'StaffUsi' in all places ? basically 'USI' to 'Usi' including file name also